learn/glossary.md

```diff
@@ -36,3 +45,13 @@ This page for instance, not being a normative document, is not able to proscribe
 #### See also
 
 * [normative](https://developer.mozilla.org/en-US/docs/Glossary/Normative) and [non-normative](https://developer.mozilla.org/en-US/docs/Glossary/non-normative) in the Mozilla Glossary, and its links
+
+### schema
+
+A document, written according to the proscribed structure of the JSON Schema specification, which can be used to validate [instances](#instance).
+
+The rules constituting what schemas are *valid* JSON Schemas, as well as the rules governing their behavior when validating instances, are defined by the JSON Schema specification.
```

Comment:
I always feel like using the term "valid" in describing schemas that don't break any rules of the specification is asking for confusing with "valid" as a result of validating an instance against a schema. Especially since a schema can be valid against the specification, or valid against its metaschema, with pretty different (though strongly related) meanings. In that suite I'm working on of schemas that violate their spec, I landed on "conformant" to the spec to differentiate that from validation.